src/SageSvg.php

```diff
@@ -78,10 +78,13 @@ public function render($image, $class = '', $attrs = [])
      */
     protected function get($image)
     {
-        if (asset($image)->exists()) {
-            return trim(
-                asset($image)->contents()
-            );
+        $manifests = app('config')->get('assets.manifests');
```

Comment:
Change this to `config('assets.manifests')` and add a line-break before the `foreach` for readability.